src/DropdownToggle.js

```diff
@@ -3,56 +3,54 @@ import classNames from 'classnames';
 import Button from './Button';
 import SafeAnchor from './SafeAnchor';
 
-import { prefix, bsClass } from './utils/bootstrapUtils';
+import { bsClass as setBsClass } from './utils/bootstrapUtils';
 
-const CARET = <span> <span className="caret" /></span>;
+const propTypes = {
+  noCaret: React.PropTypes.bool,
+  open: React.PropTypes.bool,
+  title: React.PropTypes.string,
+  useAnchor: React.PropTypes.bool,
+};
+
+const defaultProps = {
+  open: false,
+  useAnchor: false,
+  bsRole: 'toggle',
+};
 
 class DropdownToggle extends React.Component {
   render() {
     const {
-      noCaret,
-      useAnchor,
-      bsClass: bootstrapClass,
-      ...props
+      noCaret, open, useAnchor, bsClass, className, children, ...props,
```

Comment:
in general I think if we are gonna push properties to a new line we should jsut format them all on a new line...I find this style harder to read personally, not sure why.